q2cli/cli.py

```diff
@@ -0,0 +1,104 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import click
+import qiime
+from qiime.sdk import PluginManager, SubprocessExecutor
+
+from . import __version__ as q2cli_version
+
+@click.group()
+@click.pass_context
+def cli(ctx):
+    ctx.obj['plugin_manager'] = PluginManager()
+
+@cli.command()
+def version():
+    click.echo('QIIME version: %s' % qiime.__version__)
+    click.echo('q2cli version: %s' % q2cli_version)
+
+@cli.command()
+@click.pass_context
+def plugins(ctx):
+    installed_plugins = ctx.obj['plugin_manager'].plugins.items()
```

Comment:
Are the exit calls necessary here? I deferred the ``.items`` call, but I think just ``click.echo``ing should be all I need to do.